mopidy_jellyfin/frontend.py

```diff
@@ -26,11 +26,14 @@ def __init__(self, config, core):
         self.hostname = self.config['jellyfin'].get('hostname')
 
         self.wsc = WSClient(self)
-
+        self.reporting_thread = threading.Thread(target=self._check_status)
+        # Kill thread immediately on program exit
+        self.reporting_thread.daemon = True
 
     def on_start(self):
-        # Start the websocket client
+        # Start the websocket client and reporting thread
         self.wsc.start()
+        self.reporting_thread.start()
```

Comment:
can a single instance of this class receive several `on_start()`? if it can, second one would fail with an exception here that thread is already running. maybe start the thread in `__init__` and before reporting check websocket state?

Reply:
The `on_start()` and `on_stop()` functions are called from mopidy directly on program start/stop, so they shouldn't ever get called multiple times.